scionlab/tests/test_user_as_views.py

```diff
@@ -39,82 +43,71 @@ def _create_ases_for_testuser(num):
     user = get_testuser()
     num_existing_ases = UserAS.objects.filter(owner=user).count()
     for i in range(num_existing_ases, num_existing_ases+num):
-        UserAS.objects.create(
+        ap = AttachmentPoint.objects.first()
+        user_as = UserAS.objects.create(
             owner=user,
-            attachment_point=AttachmentPoint.objects.first(),
-            public_ip=_test_ip,
-            public_port=_test_start_port + i,
+            installation_type=UserAS.VM,
+            isd=ap.AS.isd,
             label="Testuser's AS number %i" % (i + 1),
-            installation_type=UserAS.VM
         )
+        ap_conf = AttachmentConf(ap,
+                                 _test_ip, _test_start_port + i,
+                                 bind_ip=None, bind_port=None,
+                                 use_vpn=False)
+        user_as.update_attachments([ap_conf])
```

Comment:
Actually we need it in the test `test_cycle_active(...)`, otherwise the `check_active(...)` method which activate/deactivates the `UserAS` won't work -- since *Activate* does nothing with no Attachments (it is just disabled).